### DISCUSSION TITLE
Header options

### DISCUSSION THREAD

**Discussion by cc-xyz** [AprilSylph/XKit-Rewritten#291] [Suggestion Box]
Hello! I hunted around for a bit for info on this, but are there any plans to import the ability to 'de-stickify' the header on desktop so that it stays fixed to the top rather than scrolling with posts? Takes up a lot of screen space on desktop and was a really useful tweak.
Thanks so much for your time! All the work you've done on the different iterations of Xkit is not only impressive but incredibly helpful, and we're all so grateful that you're continuing on with the project.
-EF

**Comment by AprilSylph** [comment]
I do actually plan to implement this under Tweaks. "Header Options" likely won't be coming back, as I don't believe I'll be reimplementing any of its other options.

---

Out of sheer curiosity, would you mind uploading a full-screen screenshot here of your Tumblr dashboard? Tumblr's header on my screen only takes up 71 pixels of a total viewport height of 937 pixels (around 7.5%), so it's intriguing to see a setup that differs enough to make it take up "a lot of screen space".

**Reply by cc-xyz** [reply]
No access to desktop right now but with my monitor it takes up about 12%. Not anything too extreme but just enough of a difference to be distracting (especially given tumblr’s issues with notification refreshes lol)

**Reply by AprilSylph** [reply]
Tumblr has no issues with notification refreshes for me. It sounds like your browser profile might be partially broken; see https://github.com/AprilSylph/XKit-Rewritten/issues/206#issuecomment-903115364 for more info.

**Reply by RobBoyBlunder** [reply]
> I do actually plan to implement this under Tweaks. "Header Options" likely won't be coming back, as I don't believe I'll be reimplementing any of its other options.

I apologize for reopening this thread but I came here to actually see if the bubble header options would make a return before making a new suggestion; I use them frequently to switch blogs on a click and am sad to hear this. I just wanted to confirm: you don't plan on bringing the blog header bubbles back in the new version then? I've been using the old xkit in tangent because of that feature. 

thanks for all your hard work on the new extension regardless!

(picture attached for context)
![blog header bubbles](https://user-images.githubusercontent.com/95882409/187335140-f04cd532-7064-4bbf-b3d9-4d5fc6c2b5be.png)

**Reply by captainharrie** [reply]
I also was wondering if there was any possibility of revisiting the blogs in header feature from header options!

The mobile app has the quick blog switcher, but there isn't really anything like that on desktop—to access my blog pages, it takes 2 clicks  (open blog list, click on posts) to reach my primary blog, and 3 (open blog list, expand blog I want to visit, click on posts) to reach my side blogs. The header options extension (when it is working) reduces that to just one click, so it's just a nice convenience tweak.

I also keep new xkit around for the blogs in header feature, but the buttons only appear with a refresh/hard load and don't appear when changing tumblr page with the built in links, so I think it'd be nice if the concept could be revisited for rewritten.
